src/govuk/components/accordion/_index.scss

```diff
@@ -42,76 +35,214 @@
+          background: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron:after {
+          color: $govuk-accordion-hover-colour;
+        }
+      }
+
+      &:focus {
+        @include govuk-focused-text;
+
+        .govuk-accordion-nav__chevron {
+          background: $govuk-accordion-base-colour;
+        }
+
+        .govuk-accordion-nav__chevron:after {
+          color: $govuk-accordion-icon-focus-colour;
+        }
+      }
+    }
+
+    .govuk-accordion__section-heading {
+      @include govuk-font($size: 24, $line-height: 1);
+      // Override browser defaults to ensure consistent element height
+      margin-top: 0; // Override browser default
+      margin-bottom: 0; // Override browser default
+    }
+
+    .govuk-accordion__section-heading-focus-wrapper {
+      margin-bottom: govuk-spacing(1);
+    }
+
+    // Create Chervon icon aligned with text
+    .govuk-accordion-nav__chevron {
+      box-sizing: border-box;
+      display: inline-block;
+
+      position: relative;
+
+      width: 20px;
+      height: 20px;
+
+      border: 1px solid;
+      border-radius: 50%;
+
+      vertical-align: text-top;
+
+      // IE8 fallback of icon
+      @include govuk-if-ie8 {
+        display: inline-block;
+        max-height: 20px;
+        line-height: 1;
+      }
+
+      // Create inner Chervon arrow
+      &:after {
+        content: "";
+        box-sizing: border-box;
+        display: block;
+
+        position: absolute;
+        bottom: 5px;
+        left: 6px;
+
+        width: 6px;
+        height: 6px;
+
+        transform: rotate(-45deg);
+
+        border-top: 2px solid;
+        border-right: 2px solid;
+
+        // IE8 fallback of icon with HTML symbol
+        @include govuk-if-ie8 {
+          content: "\2191"; // "↑"
+
+          top: 0;
+          left: 4px;
+
+          transform: none;
+
+          border: 0;
+        }
+      }
+    }
+
+    // Rotate icon to create "Down" version
+    .govuk-accordion-nav__chevron--down {
+      transform: rotate(180deg);
+
+      // IE8 fallback of arrow icon
+      @include govuk-if-ie8 {
+        &:after {
+          content: "\2193 "; // "↓"
+          transform: none;
+        }
+      }
     }
 
     // Section headers have a pointer cursor as an additional affordance
     .govuk-accordion__section-header {
-      position: relative;
-      // Safe area on the right to avoid clashing with icon
-      padding-right: 40px;
-      border-top: 1px solid $govuk-border-colour;
+      padding: 0;
       cursor: pointer;
     }
 
     // Buttons within the headers don’t need default styling
     .govuk-accordion__section-button {
-      @include govuk-typography-common;
-      margin-top: 0;
-      margin-bottom: 0;
-      margin-left: 0;
-      padding: 0;
-      border-width: 0;
-      color: $govuk-link-colour;
+      position: relative;
+
+      width: 100%;
+
+      padding: govuk-spacing(2) 0 govuk-spacing(6);
```

Comment:
Could the padding at the bottom of the button be reduced, to say `govuk-spacing(2)` to match the padding on top (10px instead of 30px)? The padding on the top of the expanded content section could then be increased slightly to maintain the spacing.

The rationale for reducing the padding would be to reduce the height that each section takes up when collapsed, as this new design is significantly taller than the current one. This might impact usability, as it is harder to scan a list of sections when you can't see as many of them at once.

The version on the [Coronavirus page](https://www.gov.uk/coronavirus) currently has 96px high sections whereas this version has 107px high sections. Existing component has 61px high sections.

Reply:
👋🏼  Hey @frankieroberto, so at this stage this current Design has been reviewed and finalised by multiple teams however the observations you've raised would be worth adding to user-testing to see the impact.

> as this new design is significantly taller than the current one. This might impact usability, as it is harder to scan a list of sections when you can't see as many of them at once.

Also, there is a [condensed version](https://components.publishing.service.gov.uk/component-guide/accordion/condensed_layout/preview) of the layout that exists for GOV.UK that might be of some interest.  I'm aware that the Accordion's guidance centres around this element being used a "last resort" and _if_ the Accordion houses too much Content then it _might_ be worth a review

Reply:
Fair enough! Will the condensed version be an option within the component?

Reply:
AFAIK not within the Design System however this option will persist on GOV.UK.  However, there _might_ be more of a case to remove this as well.

I suppose it prompts an upstream question, if there are lot of sections and the larger Design potentially impacts how a user scans  - why are there that many sections?  Does the Content need review? Also discovering the details of the scanning.. impact.
 
I've be curious to discover all the instances of this being used cross-gov to get a better understanding of use.  Equally the reserve could be true + once this is in situ / user research this might be adjusted.

Reply:
@chris-gds I suspect the accordion is used (abused?) in all sorts of different ways!

One of the primary ways I’ve seen it used though is for navigation, where each section is essentially a list of links instead of content in paragraphs.

Examples:

* https://explore-education-statistics.service.gov.uk/find-statistics
* https://www.ethnicity-facts-figures.service.gov.uk/education-skills-and-training

Perhaps there’s better ways to do this though.

Reply:
>I suspect the accordion is used (abused?) in all sorts of different ways!

+1, thanks for these - it's useful to see how it's being used elsewhere (I've not seen these before)
Out of scope to comment on this use within this PR but having more examples is useful to understand how this update might interact

Reply:
Thanks @frankieroberto @chris-gds for your thoughts!

I think you make a really good point about accordions often being misused when another component/pattern might be more suitable. We have actually have it in our backlog to update the component guidance to be clearer on this point when we publish this new version. Have a look [here](https://github.com/alphagov/govuk-design-system/issues/1684#issuecomment-846946557) (it's point 2) and feel free to leave any comments on that issue if you think there is anything else we should take into account when updating the guidance.

Reply:
@hannalaakso thanks, added a comment to that issue. Whether accordions are being misused or not is a thorny debate!

Reply:
In response - @christopherthomasdesign feedback:

> The component being taller generally is a necessary side effect of having the show all controls on a new line. I actually agree with frankie on making the padding equal on the top and bottom – need to agree this with designers on gov.uk because there was deffo a reason for it Nikin happy with 15px rather than 30 under each row

Design @nnagewad being altered to include this change.